imap_processing/run_processing.py

```diff
@@ -0,0 +1,122 @@
+"""Serves as a command line utility to run the processing for a specific
+instrument and data level.
+
+Use
+---
+    python run_processing.py <instrument> <data_level>
```

Comment:
Can you add this as an installed script as well?
```python
[project.scripts]
imap_processing = "my_package.module:function"
```

I would be explicit `imap_processing --instrument <instrument> --data-level <data_level>` Do we want data-level or just level?

Reply:
https://packaging.python.org/en/latest/specifications/declaring-project-metadata/